app/mailers/user_mailer.rb

```diff
@@ -3,7 +3,7 @@
 class UserMailer < ApplicationMailer
   SIGN_IN_EMAIL_TEMPLATE = "7ab8db5b-9842-4bc3-8dbb-f590a3198d9e"
 
-  def sign_in_email(user, url)
+  def sign_in_email(user, url, token_expiry)
```

Comment:
Should we convert this into keywords? It is hard to remember the correct order :P